guestbook/frontend-deployment.yaml

```diff
@@ -15,8 +18,10 @@ spec:
         tier: frontend
     spec:
       containers:
-      - name: php-redis
-        image: gcr.io/google-samples/gb-frontend:v4
+      - name: guestbook
+        image: paulczar/gb-frontend:v5
```

Comment:
Is this a private image? How will the image be maintained?

Reply:
The plan is to make it a non-private image – I assume there's still wok to be done, @jimangel?

Reply:
Addressed in https://github.com/kubernetes/examples/pull/410#issuecomment-861155813 and open to feedback!